doc/user_documentation/CMakeLists.txt

```diff
@@ -0,0 +1,168 @@
+find_package(Doxygen REQUIRED)
+project(dlite-doc)
+
+if(DOXYGEN_FOUND)
+
+  # Fix links in README.md such that doxygen understand them.
+  add_custom_command(
+    OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/README.md
+    COMMAND ${CMAKE_COMMAND}
+      -Dinfile=${dlite_SOURCE_DIR}/README.md
+      -Doutfile=${CMAKE_CURRENT_BINARY_DIR}/README.md
+      -P ${CMAKE_CURRENT_SOURCE_DIR}/fix_readme.cmake
+    MAIN_DEPENDENCY ${dlite_SOURCE_DIR}/README.md
+    DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/fix_readme.cmake
+    )
+
+  set(dependencies
```

Comment:
Is it documented somewhere that we have to remember to add new README or header files to this list?

Reply:
This has just been copied in from the old doc-folder. I think a better to configure doxygen and autodoc to recognize FIXME-statements or similar, to automatically generate this from the source code.